.github/workflows/ci.yml

```diff
@@ -33,4 +33,4 @@ jobs:
           set -eux
           python setup.py develop --user
       - name: Run DataPipes tests with pytest
-        run: pytest test
+        run: pytest --no-header -v test
```

Comment:
What's `--no-header`?

Reply:
No header will remove the following pattern.
![image](https://user-images.githubusercontent.com/68879799/135928928-d396ebd7-57eb-4066-bfab-78a97fe75821.png)